Library/Homebrew/test/cask/system_command_spec.rb

```diff
@@ -190,10 +189,23 @@
     end
   end
 
-  describe "given an invalid variable name" do
+  context "when given an invalid variable name" do
     it "raises an ArgumentError" do
       expect { described_class.run("true", env: { "1ABC" => true }) }
         .to raise_error(ArgumentError, /variable name/)
     end
   end
+
+  it "looks for executables in custom PATH" do
+    mktmpdir do |path|
+      (path/"tool").write <<~SH
+        #!/bin/sh
+        echo Hello, world!
+      SH
+
```

Comment:
Can we stick with the generic `<<~EOS` for now, and aim to replace it one day in a single swoop if we have decided on alternatives?

It feels a little confusing to get 98 % `EOS` (including lots of code snippets) and 2 % heredocs with other identifiers when I grep for `<<~[A-Z]+$`.

Reply:
With `SH` I get nested syntax-highlighting in TextMate.